api/token.go

```diff
@@ -572,5 +573,6 @@ func (a *API) clearCookieToken(config *conf.Configuration, name string, w http.R
 		Secure:   true,
 		HttpOnly: true,
 		Path:     "/",
+		Domain:   config.Cookie.Domain,
```

Comment:
I think that envars (all Cookie ones) need to be added to the:

* README https://github.com/supabase/gotrue/blob/master/README.md
* The GoTrue Server docs page on Supabase https://supabase.com/docs/gotrue/server/about

Also, any explanation needed in Contributing docs https://github.com/supabase/gotrue/blob/master/CONTRIBUTING.md with how to set and use for local setups?

Reply:
> The GoTrue Server docs page on Supabase https://supabase.com/docs/gotrue/server/about

hmm is there a way to keep this in-sync with the `README.md` in the repo? feels like it will be outdated over time